java/jdbcdriver-virtual-threads/src/main/java/com/oracle/dev/jdbc/PlatformThreadsOracleJdbc.java

```diff
@@ -0,0 +1,104 @@
+/*
+  Copyright (c) 2021, 2022, Oracle and/or its affiliates.
+
+  This software is dual-licensed to you under the Universal Permissive License
+  (UPL) 1.0 as shown at https://oss.oracle.com/licenses/upl or Apache License
+  2.0 as shown at http://www.apache.org/licenses/LICENSE-2.0. You may choose
+  either license.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+     https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package com.oracle.dev.jdbc;
+
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.Properties;
+import java.util.stream.IntStream;
+
+import oracle.jdbc.OracleConnection;
+import oracle.jdbc.pool.OracleDataSource;
+
+public class PlatformThreadsOracleJdbc {
+
+	private final static String DB_URL = DatabaseConfig.getDbUrl();
+	private final static String DB_USER = DatabaseConfig.getDbUser();
+	private final static String DB_PASSWORD = DatabaseConfig.getDbPassword();
+	private final static String queryStatement = "SELECT * FROM SH.CUSTOMERS WHERE CUST_ID = 49671";
+
+	public static void main(String args[]) throws SQLException {
+
+		Properties info = new Properties();
+		info.put(OracleConnection.CONNECTION_PROPERTY_USER_NAME, DB_USER);
+		info.put(OracleConnection.CONNECTION_PROPERTY_PASSWORD, DB_PASSWORD);
+		info.put(OracleConnection.CONNECTION_PROPERTY_FAN_ENABLED, false);
+		OracleDataSource ods = new OracleDataSource();
+		ods.setURL(DB_URL);
+		ods.setConnectionProperties(info);
+
+		OracleConnection connection = (OracleConnection) ods.getConnection();
```

Comment:
Would be nice to show pooled connection with UCP.
PoolDataSource pds = PoolDataSourceFactory.getPoolDataSource();

Reply:
@Kuassim done - added pooled connection with UCP for both examples (platform threads, virtual threads).